### DISCUSSION TITLE
console.log("What do I do now if I already learned HTML, CSS and JavaScript? I also have learning problems");

### DISCUSSION THREAD

**Discussion by ghost** [community/community#141902] [Programming Help]
### Body

```javascript
console.log("What do I do now if I already learned HTML, CSS and JavaScript? I also have learning problems");
```

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.
- [X] I suck at coding

**Comment by sensie01** [comment]
Learn backend framework if you like, if you not, learn Vue JS or React. (Vue JS highly recommended)

**Reply by ghost** [reply]
> Learn backend framework if you like, if you not, learn Vue JS or React. (Vue JS highly recommended)

What is Vue? I heard of React it allows you to build UIs with JavaScript.

**Reply by sensie01** [reply]
Google it bro, it's very good as react, but it's more simple and easier to learn.

Also you can use it with laravel which is very common.

**Reply by ynng3** [reply]
While I'm not sure about the situation overseas, in Korea, React is mainly used, so it seems easier to get a job with React. However, it might just be a matter of personal preference.

**Comment by Berchez** [comment]
That’s great progress! Here are some next steps you can take to continue building your skills:

1. **Learn a JavaScript Framework**: Consider learning a popular framework like React, Vue.js, or Angular. These frameworks are widely used in the industry and can help you build more complex and dynamic web applications.

2. **Backend Development**: Explore backend technologies like Node.js, Express, or Django. This will help you understand how to build and manage servers, databases, and APIs.

3. **Practice Projects**: Start working on small projects to apply what you’ve learned. This could be anything from a personal blog to a simple e-commerce site. Practical experience is invaluable.

4. **Version Control**: Learn Git and GitHub for version control. This is essential for collaborating with others and managing your codebase.

5. **Responsive Design**: Make sure your websites are mobile-friendly by learning about responsive design techniques and frameworks like Bootstrap.

6. **Accessibility**: Learn about web accessibility to ensure your websites are usable by everyone, including people with disabilities.

7. **Join Communities**: Engage with online communities like Stack Overflow, GitHub, or local meetups. These can provide support, feedback, and networking opportunities.

**Reply by ghost** [reply]
> That’s great progress! Here are some next steps you can take to continue building your skills:
>
> Learn a JavaScript Framework: Consider learning a popular framework like React, Vue.js, or Angular. 
 > These frameworks are widely used in the industry and can help you build more complex and dynamic web applications.
>
> Backend Development: Explore backend technologies like Node.js, Express, or Django. This will help you understand how to build and manage servers, databases, and APIs.
>
> Practice Projects: Start working on small projects to apply what you’ve learned. This could be anything from a personal blog to a simple e-commerce site. Practical experience is invaluable.
>
> Version Control: Learn Git and GitHub for version control. This is essential for collaborating with others and managing your codebase.
>
> Responsive Design: Make sure your websites are mobile-friendly by learning about responsive design techniques and frameworks like Bootstrap.
>
> Accessibility: Learn about web accessibility to ensure your websites are usable by everyone, including people with disabilities.
>
> Join Communities: Engage with online communities like Stack Overflow, GitHub, or local meetups. These can provide support, feedback, and networking opportunities.
>
What about a simple chat app using PHP? I just need to learn PDO.

**Comment by BhathiyaPrasad** [comment]
Make More Projects with Learned Stacks and get the fundamentals correctly
